test/wdio/computed-properties-watch-decorator/cmp.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { Fragment, h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('computed-properties-watch-decorator', function () {
+  beforeEach(async () => {
+    render({
+      template: () => (
+        <>
+          <computed-properties-watch-decorator></computed-properties-watch-decorator>
+          <button type="button">Trigger watch callbacks</button>
+        </>
+      ),
+    });
+    document.querySelector('button').addEventListener('click', () => {
+      const cmp = document.querySelector('computed-properties-watch-decorator');
```

Comment:
For my own understanding, is there any difference between this and say,
```ts
    const el = $('computed-properties-watch-decorator');
```
Would using `$` be unnecessarily `async` and add overhead to the test?

Reply:
Querying an element via `$` gives you  a `WebdriverIO.Element` instance that allows you to call WebdriverIO specific commands for it, e.g. `$('button').click()`. Now, it is not necessary to use `await` if you only use `$` since WebdriverIO only resolves the Promise behind the `$` call once you interact with the element. Basically:

```ts
// 👎 await for $ call not needed
const elem = await $('buttom')
await elem.click()
// 👍 better
await $('button').click()
```

Now, the speed improvement is marginal so both is essentially the same operation.